modules/dicom_archive/jsx/dicom_archive.js

```diff
@@ -32,7 +32,8 @@ DICOMFilterTable = React.createClass({
                     />
                     <FilterField
                         Label="Gender"
-                        Type="Text"
+                        Type="Dropdown"
+                        Options={this.props.Genders}
```

Comment:
I don't think this needs to be a property that's passed in, it can be hardcoded since the options for Gender will never change from project to project